src/interfaces/ICoWSwapOnchainOrders.sol

```diff
@@ -46,5 +46,5 @@ interface ICoWSwapOnchainOrders {
     /// @dev Event emitted to notify that an order was deleted.
```

Comment:
ubernit
Maybe we wanna not used deleted here? I think having deleted and cancelled in the wording but using invaldiation as event might be too much

Reply:
I did it [here](https://github.com/cowprotocol/ethflowcontract/pull/34). If you like the changes there I'd first merge this and that one.

Reply:
Like it all